module/zfs/metaslab.c

```diff
@@ -4789,6 +4789,34 @@ metaslab_trace_fini(zio_alloc_list_t *zal)
  * ==========================================================================
  */
 
+void
```

Comment:
Let's either add a brief comment explaining that this function is intentionally only used by `zhack`, or move this function in to `cmd/zhack.c`.